### DISCUSSION TITLE
Can't choose the collection type

### DISCUSSION THREAD

**Discussion by tmunier** [pocketbase/pocketbase#4838] [Q&A] [answered]
Hello there,
I want to create an auth collection, but I don't see any possibility to do it like explained in the docs.

<img width="1680" alt="Bildschirmfoto 2024-04-28 um 11 25 18" src="https://github.com/pocketbase/pocketbase/assets/131291552/391ab540-1df1-45f5-84df-34cae0b509de">

**Comment by ganigeorgiev** [comment, marked as answer]
Based on the screenshot it seems that you are using a very old PocketBase version (Collection types were introduced with v0.8).

Please download the latest [v0.22.10 release](https://github.com/pocketbase/pocketbase/releases).
